modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx

```diff
@@ -107,18 +126,40 @@ const ActionForm = ({
 	onSave,
 	spritemap,
 }: IFDSActionFormProps) => {
+	const ACTION_METHODS = useMemo(() => {
+		const methods = [];
+
+		for (const method in ACTION_METHOD) {
+			methods.push({label: method, value: method});
+		}
+
+		return methods;
+	}, []);
```

Comment:
Is there a reason why this is in the component? It seems to me that this should be a top level constant, sibling to `ACTION_TYPES`. With that placement, there would be no need for `useMemo`.

Reply:
I was not sure about the usage of the `useMemo` outside of the React component and basically moving it to a constant sibling to `ACTION_TYPES` should be enough.